reflex/components/core/breakpoints.py

```diff
@@ -0,0 +1,74 @@
+"""Breakpoints utility."""
+
+from typing import Optional, Tuple
+
+breakpoints_values = ["30em", "48em", "62em", "80em", "96em"]
+
+
+def set_breakpoints(values: Tuple[str, str, str, str, str]):
+    """Overwrite default breakpoint values.
+
+    Args:
+        values: CSS values in order defining the breakpoints of responsive layouts
+    """
+    breakpoints_values.clear()
+    breakpoints_values.extend(values)
+
+
+class Breakpoints(dict):
+    """A responsive styling helper."""
+
+    def __init__(self, mapping: dict):
+        """Instantiate breakpoints mapping.
+
+        Args:
+            mapping: a dictionary where breakpoints are keys and CSS values are values
+        """
+        super().__init__(mapping)
+
+    @classmethod
+    def create(
+        cls,
+        custom: Optional[dict] = None,
+        initial=None,
+        xs=None,
```

Comment:
I think we should keep this as is, but one thing that's a bit confusing is the breakpoints we size as sm, md, lg - whereas the radix core components do sizing as 1, 2, 3. I prefer the former as it's more descriptive, but there is an inconsistency.

Reply:
Radix also uses sm md lg for their responsive props, so it is consistent with them.